docs/datagrid/datagrid-v2-migration.md

```diff
@@ -0,0 +1,33 @@
+Upgrade DataGrid to v2
+===
+The core of DataGrid has been changed in the version 2.0. We increased the peformance, flexibility and added accessibility supoort. It means that API options have been changed and definition of styles or data source are different. We described all changes below. 
+
+### Data source
+In the newest DataGrid, the data source definition has been redesigned. It is a critical upgrade, so please double-check if everything is configured properly.
+
+DataGrid v1
+```js
+const grid = new DataGrid.DataGrid('container', {
+    dataTable: new DataGrid.DataTable({ columns })
+});
+```
+
+DataGrid v2
+```js
+const grid = DataGrid.dataGrid('container', {
+    table: new DataTable({
+        columns
+    })
+});
+```
+
+### Columns
```

Comment:
It's also worth pointing out that apart from the above, many more things have changed in the API. And I would give an example of a complete API config, with links to API docs or proper documentation articles.